docs/index.md

```diff
@@ -0,0 +1,339 @@
+
+## Installation
+
+### Installing via PyPI
+
+To install Aana SDK via PyPI, you can use the following command:
+
+```bash
+pip install aana
+```
+
+For optimal performance install [PyTorch](https://pytorch.org/get-started/locally/) version >=2.1 appropriate for your system. You can skip it, but it will install a default version that may not make optimal use of your system's resources, for example, a GPU or even some SIMD operations. Therefore we recommend choosing your PyTorch package carefully and installing it manually.
+
+Some models use Flash Attention. Install Flash Attention library for better performance. See [flash attention installation instructions](https://github.com/Dao-AILab/flash-attention?tab=readme-ov-file#installation-and-features) for more details and supported GPUs.
+
+### Installing from GitHub
+
+1. Clone the repository.
+
+```bash
+git clone https://github.com/mobiusml/aana_sdk.git
+```
+
+2. Install additional libraries.
+
+For optimal performance install [PyTorch](https://pytorch.org/get-started/locally/) version >=2.1 appropriate for your system. You can continue directly to the next step, but it will install a default version that may not make optimal use of your system's resources, for example, a GPU or even some SIMD operations. Therefore we recommend choosing your PyTorch package carefully and installing it manually.
+
+Some models use Flash Attention. Install Flash Attention library for better performance. See [flash attention installation instructions](https://github.com/Dao-AILab/flash-attention?tab=readme-ov-file#installation-and-features) for more details and supported GPUs.
+
+3. Install the package with poetry.
+
+The project is managed with [Poetry](https://python-poetry.org/docs/). See the [Poetry installation instructions](https://python-poetry.org/docs/#installation) on how to install it on your system.
+
+It will install the package and all dependencies in a virtual environment.
+
+```bash
+sh install.sh
+```
+
+## Integrations    
+- [Integrations](pages/integrations.md): Overview of the available predefined deployments like Whisper, vLLM, Hugging Face Transformers, Haystack etc.
+- [OpenAI API](pages/openai_api.md): Overview of the OpenAI-compatible Chat Completions API.
+
+## Deployment
+- [Docker](pages/docker.md): Instructions for using Docker with Aana SDK.
+- [Serve Config Files](pages/serve_config_files.md): Information about [Serve Config Files](https://docs.ray.io/en/latest/serve/production-guide/config.html#serve-config-files) for production deployment, how to build them, and deploy applications using them.
+- [Cluster Setup](pages/cluster_setup.md): Instructions for setting up a Ray cluster for deployment.
+
+## Configuration
+- [Settings](pages/settings.md): Documentation on the available settings and configuration options for the project.
+
+## Best Practices
+- [Code Standards](pages/code_standards.md): Learn about our coding standards and best practices for contributing to the project.
```

Comment:
The title is not good for the code standard. Best practices is more related to how to create your class and struct your app instead of contributing to the repo as a open source project

Reply:
Moved code standards to development